### DISCUSSION TITLE
How can I do this in PhpSpreadsheet?

### DISCUSSION THREAD

**Discussion by akbarali1** [PHPOffice/PhpSpreadsheet#2945] [Q&A] [answered]
[order_import_template_16072022.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/9125309/order_import_template_16072022.xlsx)
I want to generate such an excel file, how can I do it using PhpSpreadsheet?

This excel file is now created in Java

**Comment by MarkBaker** [comment]
Three worksheets, with some basic text data and some bold styling; there's nothing particularly difficult there.

Why don't you try looking at the samples and reading the documentation and writing a script to generate it yourself?

**Comment by akbarali1** [comment, marked as answer]
I did a more complicated project than the one above. If you want to see the code https://github.com/akbarali1/excel-upload-filter/blob/main/app/Http/Controllers/ExcelAutoFilterController.php


[helloWorld.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/9189763/helloWorld.xlsx)
I've uploaded an Excel file so you can see how it generated Excel
